sidebars.js

```diff
@@ -74,14 +106,27 @@ module.exports = {
               type: 'category',
               label: 'Configuring Data Docs',
               items: [
-                'guides/setup/configuring-data-docs/how-to-add-comments-to-expectations-and-display-them-in-data-docs',
-                'guides/setup/configuring-data-docs/how-to-create-renderers-for-custom-expectations',
                 'guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-a-filesystem',
                 'guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-azure-blob-storage',
                 'guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-gcs',
                 'guides/setup/configuring-data-docs/how-to-host-and-share-data-docs-on-amazon-s3'
               ]
-            }
+            },
+            {
+              type: 'category',
+              label: 'Deployment Patterns',
```

Comment:
I don't think deployment patterns belongs in the setup category. The spec seems to indicate that it should be at the same level as "How to Guides".

Reply:
I need to fix this :)